README.md

```diff
@@ -13,13 +13,12 @@
 <p align="center">
 	<a href='https://play.google.com/store/apps/details?id=phone.vishnu.quotes&pcampaignid=pcampaignidMKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1'><img alt='Get it on Google Play' src='https://play.google.com/intl/en_us/badges/static/images/badges/en_badge_web_generic.png' height="100px" /></a>
 </p>
-
 <hr>
-
-    ☆ Huge Collection of Inspirational Quotes that can be shared on Social Media Platforms as a high definition image
+<pre> 
+    ☆ Huge Collection of Inspirational Quotes that can be shared on Social Media Platforms as a<br>      high definition image
```

Comment:
Why this `<br>` tag?

Reply:
Other wise there will be horizontal scrollbar

Reply:
> Other wise there will be horizontal scrollbar

That's okay, IMO. Because, we'd have to scroll if we're on a smaller display, anyways! :D

Reply:
ok :)